app/backend/test/teet/integration/integration_email_test.clj

```diff
@@ -4,12 +4,14 @@
             [teet.test.utils :as tu]
             [clojure.string :as str]))
 
-(def outbox (atom []))
+(def outbox (atom [:server {} :msg {}]))
```

Comment:
This should be a vector of maps, each containing a `:server` and `:msg` keys.
Now the initial value is a vector of 4 values.

Reply:
so initially outbox should be empty `[]`

Reply:
Fixed.